tests/models/test_bootstrap_election_model.py

```diff
@@ -965,8 +955,8 @@ def test_get_aggregate_prediction_intervals(bootstrap_election_model, rng):
     assert lower.shape == (6, 1)
     assert upper.shape == (6, 1)
 
-    assert lower[2] == pytest.approx(upper[2])  # since c is fully reporting
-    assert lower[5] == pytest.approx(upper[5])  # since all f units are unexpected
+    assert lower[2] == pytest.approx(upper[2] - 0.002)  # since c is fully reporting
+    assert lower[5] == pytest.approx(upper[5] - 0.002)  # since all f units are unexpected
```

Comment:
I noticed here and throughout the unit tests there are these adjustments like `- 0.002`, `- 0.001`.  Why are they necessary? 🤔

Reply:
This is because I changed the code for generating lower/upper bounds so that interval_lower and interval_upper are always 0.1% away from the prediction - this is unnecessarily conservative of course when all units are reporting (but the model is never up anyways during that time).

Reply:
Ok, I see.  So this test (and the others like it) is to make sure that different is present in the lower/upper bounds.  Ok cool, thanks 😄 👍🏻